cmd/crates/soroban-rpc/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "soroban-rpc"
```

Comment:
I have pushed an empty crate to crates.io for this crate so that we know we can use it and its ready to go.